Mathlib/Topology/Algebra/InfiniteSum/Basic.lean

```diff
@@ -215,6 +215,10 @@ theorem hasSum_single {f : β → α} (b : β) (hf : ∀ (b') (_ : b' ≠ b), f
   hasSum_sum_of_ne_finset_zero <| by simpa [hf]
 #align has_sum_single hasSum_single
 
+lemma hasSum_singleton (m : β) (f : β → α) : HasSum (({m} : Set β).restrict f) (f m) := by
```

Comment:
Is there a `hasSum_unique`? That might generalize this lemma.

Reply:
There is [HasSum.unique](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/InfiniteSum/Basic.html#HasSum.unique), but it says `HasSum f a₁ → HasSum f a₂ → a₁ = a₂` , which looks a bit different.

Reply:
What would be the more general statement that you have in mind?

Reply:
(There is [hasSum_single](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/InfiniteSum/Basic.html#hasSum_single), which is used in the proof.)

Reply:
The statement would be
```lean
@[simp] lemma hasSum_singleton [Unique β] (f : β → α) : HasSum f (f default) := by
```

Reply:
OK; added.